cmd/install/addon/cmd.go

```diff
@@ -242,8 +268,51 @@ func run(cmd *cobra.Command, argv []string) {
 		})
 	}
 
+	billingModel := args.billingModel
+	billingModelAccountID := args.billingModelAccountID
+
+	if !cmd.Flags().Changed(billingModelFlag) && !interactive.Enabled() {
+		interactive.Enable()
+		r.Reporter.Infof("Enabling interactive mode")
+	}
+
+	if interactive.Enabled() && !cmd.Flags().Changed(billingModelFlag) {
+		billingModel, err = interactive.GetString(interactive.Input{
+			Question: "Billing Model",
+			Help:     cmd.Flags().Lookup(billingModelFlag).Usage,
+			Default:  "standard",
+		})
+		if err != nil {
+			r.Reporter.Errorf("Expected a valid billing model: %s", err)
+			os.Exit(1)
+		}
+	}
+
+	if !cmd.Flags().Changed(billingModelAccountIDFlag) && !interactive.Enabled() {
+		interactive.Enable()
+		r.Reporter.Infof("Enabling interactive mode")
+	}
+
+	if interactive.Enabled() && billingModel != string(v1.BillingModelStandard) &&
```

Comment:
`interactive.Enabled()` will be `true` here if `!cmd.Flags().Changed(billingModelAccountIDFlag)`